webhook_server/tests/test_repo_data_from_config.py

```
@@ -19,3 +26,101 @@ def test_repo_data_from_config_repository_found(process_github_webhook):
+        process_github_webhook._repo_data_from_config(repository_config={})
+
+    assert process_github_webhook.tox_python_version == "3.8"
+    assert "deprecated" not in caplog.text.lower()
+
+
+def test_tox_python_version_legacy_deprecation_warning(process_github_webhook, caplog):
+    """When standalone 'tox-python-version' is used instead of nested 'tox.python-version',
+    a deprecation warning should be logged."""
+    original_get_value = process_github_webhook.config.get_value
+
+    def patched_get_value(value, *args, **kwargs):
```

Ruff (0.15.13)

[warning] Missing return type annotation for private function `patched_get_value`

(ANN202)